EMBRACE_Client/DDXML/DDXMLDocument.h

```diff
@@ -0,0 +1,84 @@
+#import <Foundation/Foundation.h>
+#import "DDXMLElement.h"
+#import "DDXMLNode.h"
+
+/**
+ * Welcome to KissXML.
+ * 
+ * The project page has documentation if you have questions.
+ * https://github.com/robbiehanson/KissXML
+ * 
+ * If you're new to the project you may wish to read the "Getting Started" wiki.
+ * https://github.com/robbiehanson/KissXML/wiki/GettingStarted
+ * 
+ * KissXML provides a drop-in replacement for Apple's NSXML class cluster.
+ * The goal is to get the exact same behavior as the NSXML classes.
+ * 
+ * For API Reference, see Apple's excellent documentation,
+ * either via Xcode's Mac OS X documentation, or via the web:
+ * 
+ * https://github.com/robbiehanson/KissXML/wiki/Reference
+**/
+
+enum {
+	DDXMLDocumentXMLKind = 0,
+	DDXMLDocumentXHTMLKind,
+	DDXMLDocumentHTMLKind,
+	DDXMLDocumentTextKind
+};
+typedef NSUInteger DDXMLDocumentContentKind;
+
+@interface DDXMLDocument : DDXMLNode
+{
+}
+
+- (id)initWithXMLString:(NSString *)string options:(NSUInteger)mask error:(NSError **)error;
+//- (id)initWithContentsOfURL:(NSURL *)url options:(NSUInteger)mask error:(NSError **)error;
```

Comment:
I would like to have James' thoughts on these. Are we going to be using this methods in the future?